supervisor/backups/backup.py

```diff
@@ -161,11 +167,12 @@ def tarfile(self):
     def new(self, slug, name, date, sys_type, password=None, compressed=True):
         """Initialize a new backup."""
         # Init metadata
-        self._data[ATTR_VERSION] = 2
+        self._data[ATTR_VERSION] = 3
```

Comment:
I guess we can keep it v2 and set supervisor_version default to const.version (Schema validation) - so it's compatible to all v2 backups and because we don't have any compatibility issues, we can keep on v2 then

Reply:
Yea I was thinking about that. I just wasn't sure if the typical process was to bump the schema version for any change to the backup schema or only for backwards incompatible ones. Sounds like its only for backwards incompatible ones then so I'll revert that.